src/data/pokemon_graphics/front_pic_anims.h

```diff
@@ -1,7396 +1,1928 @@
+ANIM_BACK_AND_FORTH_2_TIMES(Tentacruel_1, 19, 35, 19, 19, 8);
+ANIM_BACK_AND_FORTH_1_TIME(Geodude_1, 15, 36, 10);
+ANIM_CONSTANT_BACK_AND_FORTH_2_TIMES(Graveler_1, 16);
+static const union AnimCmd sAnim_Golem_1[] =
 {
-    ANIMCMD_FRAME(0, 30),
-    ANIMCMD_FRAME(1, 30),
+    ANIMCMD_FRAME(0, 8),
+    ANIMCMD_FRAME(1, 31),
     ANIMCMD_FRAME(0, 1),
+    ANIMCMD_FRAME(0, 2),
+    ANIMCMD_FRAME(0, 8),
     ANIMCMD_END,
 };
-
-static const union AnimCmd sAnim_BULBASAUR_1[] =
+ANIM_2NDFRAME_1STFRAME_1_TIME(Ponyta_1, 45, 5);
+ANIM_BACK_AND_FORTH_1_TIME(Rapidash_1, 10, 40, 10);
+ANIM_BACK_AND_FORTH_1_TIME(Slowpoke_1, 20, 50, 20);
+ANIM_2NDFRAMEx2_1STFRAME(Slowbro_1, 50, 45, 10);
+ANIM_CONSTANT_BACK_AND_FORTH_2_TIMES_END(Magnemite_1, 28, 10);
+ANIM_CONSTANT_BACK_AND_FORTH_3_TIMES(Magneton_1, 14);
+ANIM_2NDFRAME_1STFRAME_6_TIMES(Farfetchd_1, 5, 5, 5, 15, 5, 5, 5, 15, 5, 5, 5, 5);
+ANIM_CONSTANT_BACK_AND_FORTH_2_TIMES_END(Doduo_1, 18, 10);
+ANIM_CONSTANT_BACK_AND_FORTH_4_TIMES_END(Dodrio_1, 12, 8);
+ANIM_2NDFRAME_1STFRAME_4_TIMES_CONSTANT(Seel_1, 15);
+ANIM_CONSTANT_BACK_AND_FORTH_1_TIME(Dewgong_1, 30);
+ANIM_CONSTANT_BACK_AND_FORTH_2_TIMES(Grimer_1, 10);
+ANIM_1STFRAME_3RDFRAME_2_TIMES_CONSTANT(Grimer_2, 20);
+ANIM_CONSTANT_BACK_AND_FORTH_2_TIMES(Muk_1, 10);
+ANIM_1STFRAME_3RDFRAME_2_TIMES_CONSTANT(Muk_2, 20);
+ANIM_2NDFRAME_1STFRAME_2_TIMES(Shellder_1, 5, 5, 45, 5);
+ANIM_2NDFRAME_1STFRAME_8_TIMES(Cloyster_1, 5, 5, 5, 5, 5, 5, 5, 25, 5, 5, 5, 5, 5, 5, 5, 5);
+ANIM_2NDFRAMEx2_1STFRAME(Gastly_1, 50, 30, 10);
+
+static const union AnimCmd sAnim_Haunter_1[] =
 {
-    ANIMCMD_FRAME(0, 30),
+    ANIMCMD_FRAME(1, 8),
+    ANIMCMD_FRAME(0, 15),
+    ANIMCMD_FRAME(1, 35),
     ANIMCMD_FRAME(1, 30),
-    ANIMCMD_FRAME(0, 1),
-    ANIMCMD_END,
-};
-
-static const union AnimCmd sAnim_IVYSAUR_1[] =
-{
-    ANIMCMD_FRAME(0, 1),
-    ANIMCMD_FRAME(1, 36),
-    ANIMCMD_FRAME(0, 10),
-    ANIMCMD_END,
-};
-
-static const union AnimCmd sAnim_VENUSAUR_1[] =
-{
-    ANIMCMD_FRAME(0, 10),
-    ANIMCMD_FRAME(1, 25),
     ANIMCMD_FRAME(0, 10),
     ANIMCMD_END,
 };
-
-static const union AnimCmd sAnim_CHARMANDER_1[] =
+ANIM_2NDFRAMEx2_1STFRAME(Gengar_1, 50, 40, 10);
+ANIM_2NDFRAMEx2_1STFRAME(Onix_1, 50, 30, 30);
+ANIM_2NDFRAME_1STFRAME_3_TIMES(Drowzee_1, 12, 12, 12, 12, 45, 10);
+ANIM_CONSTANT_BACK_AND_FORTH_3_TIMES(Hypno_1, 15);
+ANIM_2NDFRAME_1STFRAME_3_TIMES(Krabby_1, 8, 8, 8, 8, 8, 8);
+ANIM_BACK_AND_FORTH_4_TIMES(Kingler_1, 5, 15, 10, 15, 25, 10, 10, 10, 10);
+ANIM_SWITCH_BACK_AND_FORTH_2_TIMES(Voltorb_1, 4, 25);
+ANIM_SWITCH_BACK_AND_FORTH_2_TIMES(Electrode_1, 8, 30);
+ANIM_CONSTANT_BACK_AND_FORTH_4_TIMES(Exeggcute_1, 10);
+ANIM_2NDFRAME_1STFRAME_1_TIME(Exeggutor_1, 35, 10);
+ANIM_2NDFRAME_1STFRAME_3_TIMES_CONSTANT(Cubone_1, 14);
+ANIM_2NDFRAME_1STFRAME_3_TIMES_CONSTANT(Marowak_1, 14);
+ANIM_2NDFRAME_1STFRAME_1_TIME(Hitmonlee_1, 15, 20);
+ANIM_2NDFRAME_1STFRAME_4_TIMES(Hitmonchan_1, 5, 5, 5, 5, 5, 15, 25, 10);
+ANIM_BACK_AND_FORTH_1_TIME(Lickitung_1, 28, 32, 15);
```

Comment:
I appreciate that this might be something you prefer for your own workflow, but I think having 40+ macros for what are effectively arbitrary animations makes this file unexpectedly difficult to read and understand. Yes the original has more lines, but it at least looks like every other animation set up in the repo.

Reply:
What if I only submit the `gMonFrontAnimsPtrTable`, `SINGLE_ANIMATION` and `DOUBLE_ANIMATION` macros? would that be better?

Reply:
Well, honestly I did that due to the lack of consistency with other similar tables like `gMonShinyPaletteTable` where it has these macros:
```
    SPECIES_SHINY_PAL(BULBASAUR, gMonShinyPalette_Bulbasaur),
```

Reply:
Would it be better if it's made into `ANIM_CMD(BULBASAUR, sAnims_Bulbasaur),` ?

Reply:
The macro isn't doing anything useful then. If you want to list the full symbol name, just write it out normally without a macro:
```c
[SPECIES_BULBASAUR] = sAnims_Bulbasaur,
```